site/components/Home.tsx

```diff
@@ -1,29 +1,65 @@
 import Link from 'next/link';
+import { tw } from 'typewind';
 import { Features } from './Features';
 
 export default function Home() {
   return (
-    <main className="my-6 max-w-[75rem] flex flex-col px-4 mx-auto items-center">
-      <div className="w-full max-w-[53rem]">
-        <h1 className="block text-center font-extrabold text-4xl sm:text-5xl md:text-6xl mt-12">
+    <main
+      className={
+        tw.my_6.max_w_['75rem'].flex.flex_col.px_4.mx_auto.items_center
+      }
+    >
+      <div className={tw.w_full.max_w_['53rem']}>
+        <h1
+          className={
+            tw.block.text_center.font_extrabold.text_['4xl']
+              .sm(tw.text['5xl'])
```

Comment:
This line was the one causing the error. It should be `tw.text_['5xl']` instead. I have made the errors more descriptive now, so it should also report where the error occurred